src/backend/api/menu.ts

```diff
@@ -22,13 +22,8 @@ export const installEosOverlay = async () =>
   ipcRenderer.invoke('installEosOverlay')
 export const removeFromSteam = async (appName: string, runner: Runner) =>
   ipcRenderer.invoke('removeFromSteam', appName, runner)
-export const addToSteam = async (
-  appName: string,
-  runner: Runner,
-  bkgDataURL: string,
-  bigPicDataURL: string
-) =>
-  ipcRenderer.invoke('addToSteam', appName, runner, bkgDataURL, bigPicDataURL)
+export const addToSteam = async (appName: string, runner: Runner) =>
+  ipcRenderer.invoke('addToSteam', appName, runner)
```

Comment:
removed the need to pass the images from the frontend, now the backend executes some javascript in the main window instead when needed